.github/actions/init/action.yml

```diff
@@ -21,10 +21,9 @@ runs:
 
     - uses: pnpm/action-setup@v4
 
-    - uses: actions/setup-node@v4
+    - uses: actions/setup-node@v5
       with:
         node-version-file: .nvmrc
-        cache: pnpm
```

Comment:
I removed this line as [v5 implemented automatic package manager detection](https://github.com/actions/setup-node/releases/tag/v5.0.0).